ansible/files/archives/opt-custom/etc/rc-pre-network.d/010-ipf-restore.sh

```diff
@@ -62,6 +62,11 @@ update_fw() {
 		elif [[ "$(/usr/bin/svcs -Ho state ${IPF_SERVICE})" == "maintenance" ]]; then
```

Comment:
You can `return` from function if service is online and then have only one code block for clearing maintenance state.

Reply:
True. Refresh has no effect if called over service in maintenance.